Sources/Fuzzilli/Mutators/ConcatMutator.swift

```diff
@@ -12,8 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/// A mutator that concatenates two programs together.
+/// A mutator that concatenates one programs at the end of the other.
```

Comment:
Nit: I think leaving the comment as it was is fine, "concatenating" should already imply that it's inserted at the end :)

Reply:
I have reverted the comment. :)